ingest-app/src/cmr/ingest/services/bulk_update_service.clj

```diff
@@ -105,27 +106,52 @@
                                    [(format "A find value must be supplied when the update is of type %s"
                                             update-type)]))))
 
-(defn- get-provider-concept-ids
-  "Returns a list of collection concept ids for a given provider-id"
+(defn- get-provider-collection-concept-ids
+  "Returns a list of collection concept ids for a given provider-id.
+   Throws exception if non are returned."
   [context provider-id]
   (let [concepts (mdb/find-concepts context 
```

Comment:
Change this to call `mdb/find-collections`.

Reply:
Done.